.github/workflows/validate-openapi.yaml

```diff
@@ -0,0 +1,65 @@
+name: Validate OpenAPI Spec
+
+on:
+  push:
+    paths:
+      - '**/*.yaml'
+  pull_request:
+    paths:
+      - '**/*.yaml'
+
+jobs:
+  validate-openapi:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '22'
+
+      - name: Install Redocly CLI
```

Comment:
I think we can just do npm install just to make sure package.json looks good in case someone is going to update/add packages in the future

Reply:
in that case, should I include these packages `redocly and asyncapi` as devDependencies inside package.json ?

Reply:
Yes, and probably some scripts into package.json as well (`lint`, `preview` for example). Thanks Chenna!

Reply:
okay, I can do that.
But won't that increase the size of the pacakge? These tools (`redocly, asyncapi`) are only needed for the CI/CD pipeline.

If the size of the package is not a big concern, I guess there is no harm in including it inside `package.json`